style.css

```diff
@@ -1 +1,302 @@
-/*  Add your styling here */
+ /* General */
+ * {
+     box-sizing: border-box;
+ }
+
+ html,
+ body {
+     margin: 0;
+     padding: 0;
+     font-family: 'Roboto', sans-serif;
+     line-height: 1.4;
+     text-align: center;
+     background: #F6C973;
+ }
+
+ .small-font {
+     font-size: 1rem;
+     vertical-align: middle;
+ }
+
+ /* Header */
+ /* [ON SMALL SCREENS] */
+ .page-nav {
+     width: 100%;
+     height: 3.5rem;
```

Comment:
Have a look at your page and what happens to your nav-bar when you scroll down - although it stays fixed at the top, the background doesn't stick - what can you do to improve this?